nuspec/FluentNHibernate.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0'?>
+<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <id>FluentNHibernate</id>
+    <title>FluentNHibernate</title>
+    <version>0.0.0</version>
+    <description>Fluent, XML-less, compile safe, automated, convention-based mappings for NHibernate.</description>
+    <summary>Fluent, XML-less, compile safe, automated, convention-based mappings for NHibernate.</summary>
+    <authors>James Gregory and contributors (Paul Batum, Hudson Akridge, Gleb Chermennov, Jorge Rodríguez Galán).</authors>
+    <owners>jagregory, chester89, jagregory</owners>
+    <licenseUrl>https://github.com/jagregory/fluent-nhibernate/raw/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/jagregory/fluent-nhibernate</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/jagregory/fluent-nhibernate/master/docs/logo-nuget.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <tags>ORM DAL NHibernate DataBase ADO.Net Mappings Conventions</tags>
+    <copyright>Copyright (c) James Gregory and contributors (Paul Batum, Hudson Akridge, Gleb Chermennov, Jorge Rodríguez Galán).</copyright>
+    <dependencies>
+  		<dependency id="NHibernate" version="5.0.3" />      

```

Comment:
Might be better to use this this syntax [5.0.3, 6.0)

Reply:
You're right!

Reply:
I don't think that would affect builds since the specific version of NHibernate will be specified in the projects packages.config anyway. I think it will only affect the version that is initially installed into a project the first time someone adds FluentNHibernate.

I am not sure if specifying the upper limit on the NHibernate version is a good idea or not as we don't know if version 6.0 will break any functionality with fluent.